packages/ui/src/ui/components/ACL/ColumnGroups/ColumnGroups.tsx

```diff
@@ -25,9 +25,16 @@ interface Props {
     path: string;
     loadAclDataFn: () => void;
     cluster: string;
+    hasCreateForm?: boolean;
 }
 
-export default function ColumnGroups({columnGroups, path, loadAclDataFn, cluster}: Props) {
+export default function ColumnGroups({
+    columnGroups,
+    path,
+    loadAclDataFn,
+    cluster,
+    hasCreateForm = false,
```

Comment:
It looks like editing is still allowed for all node types.
Will be an edited item applied only for current path?
Do we need to allow edit only for `map_nodes`?

Reply:
by now, we are expecting to have column groups only for 'map_nodes'

i've asked the team whether should i remove the whole Column groups component for other node types, but they asked not to do it right now. They have some plans for it (maybe in some future, they will provide smth like @effective_column_groups attribute to display it for tables and other child nodes)

So i think, we are safe not to worry about editing, because user will be not allowed to create column groups for that nodes -> so the table will be empty

Reply:
I changed my mind)

I am going to hide actions column (delete and edit) for other node types
Also will rename `hasCreateForm` prop to `allowEditing` or smth more suitable to handle both: hiding Add+ button and action buttons